dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj

```diff
@@ -8,9 +8,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.2.0" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.2.1" />
   </ItemGroup>
-  
+
+  <!-- For consistency across Reunion, explicitly reference .NET 5.0.5 SDK (5.0.202 train for VS 16.9.3) -->
+  <ItemGroup>
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.18362.15" />
```

Comment:
>15 [](http://example.com/codeflow?start=99&length=2)

probably ok for cherry pick. but this needs to be 16, or i'd rather remove this altogether.

Reply:
15 to both of these I assusme? Whats the reason for removing this altogether?

Reply:
I believe referencing to cswinrt 1.2.1 will bring it to 16.

Reply:
It does that automatically?

Reply:
C#/WinRT 1.2.2 brings in the version of winrt.runtime.dll included in the *.16 framework package:
https://github.com/microsoft/CsWinRT/releases/tag/1.2.2.210413.1

Reply:
Project Reunion: build pipelines should assert version consistency of shared dependencies    
https://microsoft.visualstudio.com/OS/_workitems/edit/32797522

Reply:
Updated to 16

Reply:
then the comment is wrong now. i think 16 maps to .net 5.0.6

Reply:
Oh shoot, didn't realize my update is to 1.2.1. Should I just update cswinrt to 1.2.2 so it is in sync?

Reply:
I'd leave this question to @Scottj1s. WinUI is moving to 1.2.5. :)

Reply:
The only essential compat consideration is the AssemblyVersion of winrt.runtime.dll.  C#/WinRT 1.2.2 bumped winrt.runtime.dll to AssemblyVersion 1.2.0.0, and it hasn't changed since then.  So 1.2.2 ... 1.2.5 are all compatible.

Reply:
according to https://github.com/microsoft/CsWinRT/releases, the AssemblyVersion was bumped to 1.2.0.0 since 1.2.0. 
@kythant  the comment is still incorrect.

Reply:
You are right, Hui - 1.2.0 ... 1.2.5 are compatible

Reply:
wow, codeflow now understands github threading.

Reply:
if only github understood codeflow comments (with row/column metadata)